data/sql/updates/pending_db_auth/rev_1751147997826401300.sql

```diff
@@ -0,0 +1,3 @@
+--
+ALTER TABLE `autobroadcast_locale`
+    CHANGE COLUMN `text` `text` LONGTEXT NOT NULL COLLATE 'utf8mb4_unicode_ci' AFTER `locale`;
```

Comment:
longtext

Reply:
It matches the autobrodcast it's LONGTEXT being used